core/src/test/java/org/elasticsearch/search/sort/ScriptSortBuilderTests.java

```diff
@@ -58,4 +60,13 @@ public void testToXContentWithParams() throws Exception {
         
         assertEquals(xcWith.string(), xcWithOut.string());
     }
+    
+    public void testToString() {
+        ScriptSortBuilder ssb = prepareSSB();
+
+        String ssbPrint = ssb.toString();
+
+        assertThat(ssbPrint, containsString("testtype"));
```

Comment:
Same here, simple assertQuals with String comparison would be easier to read I think.